services/discovery/v1-query-builder.js

```diff
@@ -61,17 +75,13 @@ module.exports = function(RED) {
                                          sApikey ? sApikey : req.query.key,
                                          req.query.endpoint ? req.query.endpoint : sEndpoint);
 
-    discovery.listCollections({
-      environment_id: req.query.environment_id
-    },
-      function(err, response) {
-        if (err) {
-          res.json(err);
-        } else {
-          res.json(response.collections ? response.collections : response);
-        }
-      }
-    );
+    discovery.listCollections({environmentId: req.query.environment_id})
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4846228)